diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -439,17 +444,23 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
      *
      * @return true - if there is error, and false if there is no error
      */
-    private fun ASTNode.isCallsChain() = getCallChain()?.isNotValidCalls(this) ?: false
-
-    private fun ASTNode.getCallChain() = getParentExpressions()
-        .lastOrNull()
-        ?.run {
-            mutableListOf<ASTNode>().also {
-                getOrderedCallExpressions(psi, it)
+    private fun ASTNode.isCallsChain(isWithoutParentheses: Boolean = true) = getCallChain(isWithoutParentheses)?.isNotValidCalls(this) ?: false
```

Comment:
What is the meaning of `isWithoutParentheses` parameter? If it is intended to get all calls without dropping first ones, which we don't count for functional-style-check, it should be called differently.